Sources/Shared/Configuration.swift

```diff
@@ -110,6 +110,9 @@ public final class Configuration {
     @Setting(key: "relative_results", defaultValue: false)
     public var relativeResults: Bool
 
+    @Setting(key: "json_package_manifest_path", defaultValue: nil)
+    public var jsonPackageManifestPath: String?
```

Comment:
Can you also people add the references in `asYaml`, `load` and `reset` like the other config options?

Reply:
Sure! I made a new commit with changes.